peerscout_api/recommend_editor.py

```diff
@@ -125,5 +125,4 @@ def get_editor_recommendations_for_api(
         for predicted_editor in row
     ])
 
-    return prediction_results_flat_df.sort_values(
-        'score', ascending=False).head(top_n_editor)
+    return prediction_results_flat_df.head(top_n_editor)
```

Comment:
Are the editors already sorted?

Reply:
Yes, additionally it is been using in notebooks like that.

Reply:
For the notebooks it may not be important that they are sorted because it is sorted by Data Studio.
But you are right, there sorting is part of `get_recommended_editors_with_probability`